frontend/src/components/rider-table.tsx

```diff
@@ -0,0 +1,348 @@
+                <label htmlFor="accesibility"> Has Crutches</label>
+              </div>
+              <div className="checkboxDiv">
+                <input type="checkbox" id="accesibility" name="needAssist"
+                  onChange={(e) => handleInput(e)}
+                />
+                <label htmlFor="accesibility"> Needs Assistant</label>
+              </div>
+            </div>
+          </div>
+          <div className="formDiv">
+            <label htmlFor="description" className="formLabel">
+              Description:
+          </label >
+            <input type="text"
+              name="description"
+              onChange={(e) => handleInput(e)} />
+            <p className={`formFeedback ${validDesc ? "hidden" : ""}`}>
+              Enter a description
+          </p>
+          </div>
+          <div className="formDiv">
+            <label htmlFor="pronouns" className="formLabel">Pronouns: </label >
+            <select name="pronouns" onChange={(e) => handleInput(e)}>
+              <option value="she">She</option>
+              <option value="he">He</option>
+              <option value="neutral">Neutral</option>
+            </select>
+          </div>
+          <div className="formDiv">
+            <label htmlFor="address" className="formLabel">Address: </label >
+            <input type="text"
+              name="address"
+              onChange={(e) => handleInput(e)} />
+            <p className={`formFeedback ${validAddress ? "hidden" : ""}`}>
+              Enter an address
+          </p>
+          </div>
+          <input type="submit" value="Submit" />
+        </form>
+      </div>
+    </>
+  );
+};
+
+const Table = () => {
+  const [riders, setRiders] = useState([
+    {
+      firstName: 'first1', lastName: 'last1', phoneNumber: '111-111-1111',
+      email: 'a1@cornell.edu', accessibilityNeeds:
+        { needsWheelchair: true, hasCrutches: true, needsAssistant: false },
+      description: 'descr1', joinDate: 'join1', pronouns: 'pro1', address: 'add1'
+    },
+    {
+      firstName: 'first2', lastName: 'last2', phoneNumber: '222-222-2222',
+      email: 'b2@cornell.edu', accessibilityNeeds:
+        { needsWheelchair: false, hasCrutches: true, needsAssistant: false },
+      description: 'descr2', joinDate: 'join2', pronouns: 'pro2', address: 'add2'
+    }
+  ]);
+
+  function renderAccessNeeds(accessNeeds: AccessibilityNeeds) {
+    let allNeeds = '';
+    let arrayNeeds = Object.entries(accessNeeds);
+    arrayNeeds.forEach(element => {
+      if (element[1]) {
+        if (element[0] === 'needsWheelchair') {
+          allNeeds = allNeeds.concat("Needs Wheelchair, ");
+        } else if (element[0] === 'hasCrutches') {
+          allNeeds = allNeeds.concat("Has Crutches, ");
+        } else {
+          allNeeds = allNeeds.concat("Needs Assistant, ");
+        }
+      }
+    });
+    return allNeeds.substr(0, allNeeds.length - 2);
+  }
+
+  function renderTableData(allRiders: Rider[]) {
```

Comment:
Unsure where they are exactly, but resolve the errors that appear in the browser console. You should be able to log in fine now.